htlcswitch/test_utils.go

```diff
@@ -449,7 +449,7 @@ func (n *threeHopNetwork) makePayment(sendingPeer, receivingPeer Peer,
 	select {
 	case err := <-errChan:
 		return invoice, err
-	case <-time.After(5 * time.Minute):
+	case <-time.After(15 * time.Second):
```

Comment:
When running with the `-race` flag on Travis, I've seen this case take up to `1 minute` to complete, which is why I had the timeout set to 5. Normally it completes very quickly, but Travis is a different beast...

Reply:
Do'h! reverting then 🤠